root/Makefile

```
@@ -14,7 +14,7 @@ LDFLAGS             ?= -w -s -X github.com/getoutreach/gobox/pkg/app.Version=$(A
 GOFLAGS             :=
 GOPRIVATE           := github.com/$(ORG)/*
 GOPROXY             := https://proxy.golang.org
-GO_EXTRA_FLAGS      := -v -tags=or_dev
+GO_EXTRA_FLAGS      := -v $(shell [[ ! -z "$KUBERNETES_SERVICE_HOST" ]] && printf "-tags=or_dev")
```

nit `! -z` -> `-n`

---

```suggestion
GO_EXTRA_FLAGS      := -v $(shell [[ -n "$KUBERNETES_SERVICE_HOST" ]] && printf "-tags=or_dev")
```

---

Wait, we want `-z` here, right? If `KUBERNETES...` is empty, use `or_dev`. Not the other way around?

---

or am I misreading the one-liner, I hate bash one-line if statements

---

this seems to do what we want, if `KUBERNETES...` is empty, add the `or_dev`. I hate bash.

---

Hmmmm, ok new suggestion (fun fact don't need to quote in if statement)

ref: https://stackoverflow.com/questions/18096670/what-does-z-mean-in-bash

```suggestion 
GO_EXTRA_FLAGS      := -v $(shell if [[ -z $KUBERNETES_SERVICE_HOST ]]; then printf "-tags=or_dev"; fi)
```